app/controllers/additional_controller.rb

```diff
@@ -17,11 +17,13 @@ def ads
   def uploader
     @urls = Rails.cache.fetch('uploader_url', expires_in: 2.days) do
       urls = Hash.new
-      git_response = HTTParty.get('https://api.github.com/repos/HearthStats/HearthStats.net-Uploader/releases?per_page=1', headers: { "User-Agent" => "HearthStats"})
-      urls["osx"] = git_response[0]["assets"][0]["browser_download_url"]
-      urls["windows"] = git_response[0]["assets"][1]["browser_download_url"]
+      win_git_response = HTTParty.get('https://api.github.com/repos/Epix37/Hearthstone-Deck-Tracker/releases?per_page=1', headers: { "User-Agent" => "HearthStats"})
```

Comment:
Line is too long. [164/80]<br>Space inside } missing.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.